### PR TITLE
Adding notes to emergency removal cmdlets

### PR DESCRIPTION
**Pull request by JensTrier**
Adding notes to the two removal cmdlets and some formatting changes.

### CODE REVIEW

File: skype/skype-ps/skype/Remove-CsOnlineLisLocation.md [hunk 1]
```diff
@@ -13,7 +13,10 @@ ms.reviewer:
 # Remove-CsOnlineLisLocation
 
 ## SYNOPSIS
-Use the \`Remove-CsOnlineLisLocation\` cmdlet to remove an existing emergency location from the Location Information Service (LIS.) You can only remove locations that have no assigned users.
+Use the Remove-CsOnlineLisLocation cmdlet to remove an existing emergency location from the Location Information Service (LIS.)
```
Comment (JohanFreelancer9) [inline skype/skype-ps/skype/Remove-CsOnlineLisLocation.md:16]:
```suggestion
Use the Remove-CsOnlineLisLocation cmdlet to remove an existing emergency location from the Location Information Service (LIS). 
```

**Review by JohanFreelancer9 — CHANGES_REQUESTED** [review]
Copy review - @JensTrier, minor additional edit, please.

**Comment by JensTrier** [comment]
> Copy review - @JensTrier, minor additional edit, please.

Thanks @JohanFreelancer9! Done.

**Review by JohanFreelancer9 — APPROVED** [review]

**Review by get-itips — APPROVED** [review]

**Review by dariomws — APPROVED** [review]

**Comment by yogkumgit** [comment]
@tiburd Please merge this PR. Thanks

**Pull request merged**